packages/babel-traverse/test/family.js

```diff
@@ -52,6 +52,28 @@ describe("path/family", function () {
         assert.strictEqual(outerNodes[id], outerPaths[id].node, "nodes match");
       });
     });
+
+  });
+  describe("getSibling", function () {
+    const ast = parse("var a = 1, {b} = c, [d] = e; function f() {}");
+    let sibling = {};
+    traverse(ast, {
+      VariableDeclaration(path) {
+        sibling = path.getSibling(path.key);
+      }
+    });
+
+    it("should return traverse sibling nodes", function () {
+      assert.ok(sibling.getNextSibling().node, "has property node");
+      assert.ok(sibling.getNextSibling().getPrevSibling().node, "has property node");
+      assert.equal(!!sibling.getPrevSibling().node, false, "out of scope");
+      assert.equal(!!sibling.getNextSibling().getNextSibling().node, false, "out of scope");
+    });
+
+    it("should return all preceding and succeeding sibling nodes", function () {
+      assert.ok(sibling.getAllNextSiblings().length, "Has next sibling");
+      assert.ok(sibling.getNextSibling().getAllPrevSiblings().length, "Has prev sibling");
```

Comment:
That makes sense. Need to make sure the method returns the correct number of siblings.